pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -140,12 +140,22 @@ func (r *CPUSuppress) calculateBESuppressCPU(node *corev1.Node, nodeMetric *metr
 		systemUsedCPU = *resource.NewMilliQuantity(0, resource.DecimalSI)
 	}
 
-	// suppress(BE) := node.Total * SLOPercent - pod(LS).Used - system.Used
+	notPodUsed := systemUsedCPU
+	rl := util.GetNodeReservationFromAnnotation(node.Annotations)
+	if rl != nil {
+		nodeAnnoReserved, _ := rl[corev1.ResourceCPU]
```

Comment:
also use `quotav1.Max()`

Reply:
type of systemUsedCPU if `quantity`, but Max(a v1.ResourceList, b v1.ResourceList)

Reply:
done